pkg/execution/state/redis_state/key_generator.go

```diff
@@ -3,24 +3,22 @@ package redis_state
 import (
 	"context"
 	"fmt"
+	"time"
 
 	"github.com/google/uuid"
 	osqueue "github.com/inngest/inngest/pkg/execution/queue"
 	"github.com/inngest/inngest/pkg/execution/state"
 	"github.com/oklog/ulid/v2"
 )
 
-var (
-	defaultQueueKey = DefaultQueueKeyGenerator{}
-)
+// TODO Set this properly, don't start sharding any time soon before we are sure everything works
+var switchover = time.Date(2100, 07, 10, 0, 0, 0, 0, time.UTC)
```

Comment:
This is set to a fixed future value for everyone right now. When moving this into production, we should determine the value for specific user groups.